src/args.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"fmt"
+
+	sdkArgs "github.com/newrelic/infra-integrations-sdk/args"
+	"github.com/newrelic/infra-integrations-sdk/log"
+)
+
+func getDbAndKeys(keysFlag sdkArgs.JSON) map[string][]string {
+	databaseKeys := make(map[string][]string)
+
+	convertF := func(listInterface []interface{}) []string {
+		listString := make([]string, 0)
+		for _, elem := range listInterface {
+			if _, ok := elem.(string); ok {
+				listString = append(listString, elem.(string))
+			} else {
+				log.Warn("Not expected type for key: %v, string type required", elem)
+			}
+		}
+		return listString
+	}
+
+	switch source := keysFlag.Get().(type) {
+	case []interface{}:
+		databaseKeys["0"] = removeDuplicates(convertF(source))
+	case map[string]interface{}:
+		for db, keys := range source {
+			databaseKeys[db] = removeDuplicates(convertF(keys.([]interface{})))
+		}
+	default:
+		log.Warn("Invalid format, value of keys flag: %v ", keysFlag)
+	}
+	return databaseKeys
+}
+
+// getRenamedCommands returns a map containing command and renamed command pairs
+// Example flag value: '{"CONFIG":"ZmtlbmZ3ZWZl-CONFIG"}'
+func getRenamedCommands(renamedCommandsFlag sdkArgs.JSON) (map[string]string, error) {
+	renamedCommands := make(map[string]string)
+
+	convertF := func(stringInterface interface{}) (string, error) {
+		str := ""
+		if _, ok := stringInterface.(string); ok {
+			str = stringInterface.(string)
+		} else {
+			return "", fmt.Errorf("Not expected type for key: %v, string type required", stringInterface)
+		}
+		return str, nil
```

Comment:
We can simplify this a bit by removing the else :)

```suggestion
		if str, ok := stringInterface.(string); ok {
			return str, nil
		}

		return "", fmt.Errorf("unexpected type %T, value must be a string", stringInterface)
```